### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by trungdlp-wolffun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,8 +40,8 @@ The Mongo ODM for Go
 - `mgm` wraps the official Mongo Go Driver.
 
 ## Requirements
-- Go 1.10 or higher.
-- MongoDB 2.6 and higher.
+- Go 1.17 or higher.
+- MongoDB 4.2 and higher.
```
Comment (mehran-prs) [inline README.md:44]: Thanks for your contribution @trungdlp-wolffun :))
You've updated the monog-driver's version to `v1.10.3` in `go.mod` file, that version of Mongo official driver [supports](https://github.com/mongodb/mongo-go-driver/tree/v1.10.3) MongoDB 2.6 or higher, so actually we support MongoDB 2.6 or higher, unless you update the mongo driver's version to its latest version and then change this line to [MongoDB 3.6 and higher](https://github.com/mongodb/mongo-go-driver/tree/master#requirements)

**Comment by mehran-prs** [comment]
@trungdlp-wolffun sorry to say that I missed your PR and  just noticed it right now (by the PR's closement notification),
We'll update this library's dependencies.

**Comment by trungdlp-wolffun** [comment]
@mehran-prs please take a look 🥺

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (trungdlp-wolffun) [reply README.md:44]: okay, I will update soon

[the remark this answers is not in the document]
Comment (trungdlp-wolffun) [reply README.md]: I updated @mehran-prs

**Comment by trungdlp-wolffun** [comment]
Hey @mehran-prs, how do I run unit tests? Do you have any CI in place to verify them?

**Review by mehran-prs — COMMENTED** [review]
LGTM

**Comment by mehran-prs** [comment]
@trungdlp-wolffun our build system was travis ci which is disabled temporarily and we'll migrate to Github actions.
but I've tested your changes locally and it passes the tests, I'll merge your changes into the master branch, but after adding Github actions and testing on all supported Go versions I'll tag your changes and release them.

Thanks for your contribution and for upgrading its dependencies versions ❤️.

**Pull request merged**